mkt/search/filters.py

```diff
@@ -47,16 +47,23 @@ def filter_queryset(self, request, queryset, view):
             rules.append(
                 (query.Fuzzy, {'value': q, 'boost': 2, 'prefix_length': 1}))
 
+        # Apply rules to search on few base fields. Some might not be present
+        # in every document type / indexes.
         for k, v in rules:
-            for field in ('name', 'app_slug', 'author'):
+            for field in ('name', 'title', 'short_title',
+                          'app_slug', 'author', 'url'):
                 should.append(k(**{field: v}))
 
         # Exact matches need to be queried against a non-analyzed field. Let's
-        # do a term query on `name.raw` for an exact match against the app
+        # do a term query on `name.raw` for an exact match against the item
         # name and give it a good boost since this is likely what the user
         # wants.
         should.append(query.Term(**{'name.raw': {'value': q, 'boost': 10}}))
 
+        # FIXME: for websites, what should we match against ? "title" is going
+        # to be extracted from websites <title> so it won't work. "short_title"
+        # is only intended for display purposes. Or should we re-purpose it ?
```

Comment:
For now we decided to keep the `title` and add a `title.raw` even though on some websites we'll never get the exact match. We might revisit this later and add a separate `name` attribute.